performant-software/Neatline
# Changed ol2 config to allow mapwarper WMS maps as base layers in exhibits

### What does this PR do?
- Hardcodes Web Mercator as the map projection to use, which fixes issues with WMS, per openlayers/ol2#350, which addresses #391
- Changes how Neatline deals with WMS layers in exhibits, which addresses #358 
- Also possibly related to: #403, which I can't currently reproduce; that may have been a side effect of specific version of Chrome and specific version of the server at mapwarper.net, as noted [here](https://github.com/scholarslab/Neatline/issues/403#issuecomment-292587931)

### How to test
- Go to http://neatline-staging.herokuapp.com/admin/neatline/edit/2, note that there are both WMS map address and a WMS layer specified.
- Go to: http://neatline-staging.herokuapp.com/neatline/show/london, and note that that WMS layer does load, in addition to the other spatial layers.
